src/chart/CirclePackagingChart.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ChartProps } from './Chart';
+import CirclePackagingVisualization from './visualizations/CirclePackagingVisualization';
+
+
+/**
+ * Embeds a CirclePackagingChart (from Charts) into NeoDash.
+ */
+const NeoCirclePackagingChart = (props: ChartProps) => {
```

Comment:
Shouldn't this be 'Circle Packing'?

Reply:
U r right! There r a couple modifications to here then in order to adjust the naming.